src/Dashboard/Provider.php

```diff
@@ -801,7 +801,7 @@ public static function nbItemByFk(
             'label'           => "",
             'searchoption_id' => $found_so_id,
             'icon'            => $fk_item::getIcon() ?? $item::getIcon(),
-            'limit'           => 50,
+            'limit'           => 99999,
```

Comment:
If the aim is not to limit the results, the best thing to do is not to define a LIMIT and adapt the function so that it also works without a LIMIT.
```suggestion
```